packages/fiber/src/core/events.ts

```diff
@@ -201,9 +205,16 @@ export function createEvents(store: UseStore<RootState>) {
           stopPropagation: () => {
             // https://github.com/pmndrs/react-three-fiber/issues/596
             // Events are not allowed to stop propagation if the pointer has been captured
-            if (!('pointerId' in event && internal.capturedMap.has(event.pointerId))) {
-              raycastEvent.stopped = localState.stopped = true
 
+            // We only authorize stopPropagation...
+            if (
+              // ...if the event doesn't have a pointerId (hence it can't
+              // have been captured)...
+              !('pointerId' in event) ||
+              // ... or if the hit object is capturing the event
+              internal.capturedMap.get(event.pointerId)?.has(hit.eventObject)
```

Comment:
I've just come to run this code and I've noticed that this logic is wrong now. When the `pointerId` isn't captured by anything, this line will evaluate to `undefined` but it needs to be `true` in that case so that you can `stopPropagation`.

Reply:
Arg right...